src/openfermion/hamiltonians/_molecular_data.py

```diff
@@ -820,16 +820,16 @@ def get_active_space_integrals(self,
                                              active_indices,
                                              active_indices)])
 
-    def get_molecular_hamiltonian(self,occupied_indices=None,
-                                  active_indices=None):
+    def get_molecular_hamiltonian(self, occupied_indices=None,
+                                      active_indices=None):
```

Comment:
you had it right the first time actually! It should align under self.

Reply:
i'm gonna get it all right next time!!